src/lib/holocene/modal.svelte

```diff
@@ -21,45 +25,82 @@
     dispatch('cancelModal');
   };
 
-  const handleKeyUp = (event: KeyboardEvent) => {
+  const handleKeyboardNavigation = (event: KeyboardEvent) => {
+    if (!open) {
+      return;
+    }
+
     if (event.key === 'Escape') {
       cancelModal();
+      return;
+    }
+
+    if (event.key === 'Tab') {
+      if (event.shiftKey) {
+        if (document.activeElement === closeButton) {
+          confirmButton.focus();
+          event.preventDefault();
+        }
+      } else if (document.activeElement === confirmButton.buttonElement) {
+        closeButton.focus();
+        event.preventDefault();
+      }
     }
   };
+
+  $: {
```

Comment:
I think we can utilize `bind:this` and focus the modal instead of the button (which will make the screen reader a little bit easier to understand and avoids `autofocus`).
```suggestion
  $: {
    if (open && modal) {
      modal.focus();
    }
  }
```

 See additional comment below 👇